### DISCUSSION TITLE
v1.11.0

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#586] [Announcements]
⚠️  **Support for PHP 7.4 was dropped. LinkAce only supports PHP 8.0, 8.1 and 8.2.**

- Added the Italian translation provided by Alessandro Del Prete
- Added the Polish translation provided by Mateusz Hajder and Yonei
- QR Codes for 2-Factor-Authentication (2FA) can be scanned when dark mode is activated. ([#501](https://github.com/Kovah/LinkAce/issues/501))
- Toggles for active share services are now properly handled. ([#581](https://github.com/Kovah/LinkAce/issues/581))
- The “Links are private by default” setting is taken into account when importing bookmarks. ([#577](https://github.com/Kovah/LinkAce/issues/577))
- Updated dependencies: Laravel 9, Spatie Backup and many more
- LinkAce now offers a bug bounty program for critical security vulnerabilities. Details can be found in the [Security Policy](https://github.com/Kovah/LinkAce/security/policy).

<hr /><em>This discussion was created from the release <a href='https://github.com/Kovah/LinkAce/releases/tag/v1.11.0'>v1.11.0</a>.</em>

**Comment by martadinata666** [comment]
v1.11.0 missing release zip files? 🤔 
![2022-12-16-095337_867x887_scrot](https://user-images.githubusercontent.com/2433562/208011397-83e3f61b-d091-4806-b777-9bbb8e3c6c30.png)

**Reply by Kovah** [reply]
Oh. 😆 
Seems I missed that. Added the missing packages. For those who are curious: they are built in the "[Build Release Package](https://github.com/Kovah/LinkAce/actions/workflows/build-package.yml)" action, which can be accessed from the Actions tab.

**Comment by alexdelprete** [comment]
Thanks for the credits. :)
